### DISCUSSION TITLE
Goes to recovery after UTM logo is seen

### DISCUSSION THREAD

**Discussion by ML1998** [utmapp/UTM#3030] [General]
![Screen Shot 2021-08-15 at 10 39 11 am](https://user-images.githubusercontent.com/88952116/129463486-a9006e0b-832f-4176-b916-ca4240b82985.png)

**Comment by muratdk23** [comment]
You installing from vhdx image?

**Comment by ML1998** [comment]
I’ve converted it to I forgot what it was but did as suggested by UTM. But
the strange thing is converted file was 69GB while original was 9 GB.
Couldn’t figure out what it was.

**Comment by muratdk23** [comment]
It's normal. Real size is 9 GB on disk. I converted image using this command "qemu-img convert filename.vhdx filename.qcow2". You used this command?

**Comment by ML1998** [comment]
Yes I did the same thing. Converted to qcow2 using same command.

**Comment by conath** [comment]
I am moving this to Discussions since no technical information is provided and this doesn't look like a UTM problem.
